releng/com.zeligsoft.papyrus-cx.axcioma.rcp/about.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```

Comment:
CX is Apache 2.0 licensed, not EPL. The Apache License text is in the root of the project